avm/res/data-factory/factory/main.bicep

```
@@ -63,6 +63,9 @@ param gitTenantId string = ''
 @description('Optional. List of Global Parameters for the factory.')
 param globalParameters object = {}
 
+@description('Optional. Purview Account resource identifier.')
+param purviewResourceId string = ''
```

```suggestion
param purviewResourceId string?
```

`empty(null)` also translates to `false`. Maybe it's just me, but I feel nullable is a bit 'cleaner' than an empty string as a default for an optional parameter. Feel free to disagree though 😄